### DISCUSSION TITLE
Can deepmd-kit help me normalize the label values?

### DISCUSSION THREAD

**Discussion by yyyien** [deepmodeling/deepmd-kit#1889] [Q&A] [answered]
My force is very small, will my label value be normalized?Can deepmd-kit help me normalize the label values?

**Comment by wanghan-iapcm** [comment, marked as answer]
No, you have to do it manually.

**Reply by yyyien** [reply]
OK,thank you very much.

**Comment by njzjz** [comment]
Forces MUST be the negative gradients of the energy.
